results.ja.html

```
@@ -0,0 +1,734 @@
+    <div class="verdicts" id="leverList"></div>
+    <p class="note">各判定はそれぞれ事前登録したルールを持つ本物のスイープです。「逆効果」と「無駄」は本物の否定的結果で、こっそり捨てずに記録に残しています。</p>
+  </div>
+</section>
+
+<!-- 06 resolution -->
+<section class="reveal">
+  <div class="wrap">
+    <div class="shead"><span class="snum">06 / 10</span><span class="skick">解決</span><span class="bar"></span></div>
+    <h2><span class="hicon"><svg class="ic" aria-hidden="true" focusable="false"><use href="#trend"/></svg></span>失敗は設計ではなく用量だった</h2>
+    <p class="lead">三人目が加わると、各エージェントに番が回る頻度が下がり、同じコストでも住人が増えるぶん一人への圧力は弱まります。直したのは賢い設計ではなく、ただ<strong>用量を大きな村に合わせた</strong>こと(contribute コストを 30 から 42 へ)。そうすると三人の村はラインを越えます。ダイバージェンスは <strong>0.21 から 0.31</strong> へ上がり、新しいシード3つすべてで合格しました。<a class="cite" href="https://github.com/TheIllusionOfLife/microverse/blob/main/docs/adr/0018-action-economy-r3-levers2.md" target="_blank" rel="noopener">ADR&nbsp;0018</a></p>
+    <div class="callout win"><b>誠実な但し書き:</b> これは<em>救済された三人の村が一つ</em>であって、証明された法則ではありません。単純な「希釈」の説明 — 大きな村には大きな用量 — の良い証拠で、3シードすべてで分業の基準を、さらに厳しい役割安定性チェックも3つ中2つで越えました。ただし、はるかに大きな集団・別の行動メニュー・別のモデルでも成り立つことは、<em>まだ</em>示せていません。</p></div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </p> ]

(tag-pair)